libfastpm/store.c

```diff
@@ -191,6 +191,7 @@ fastpm_store_init_details(FastPMStore * p,
     DEFINE_COLUMN(acc, COLUMN_ACC, "f4", 3);
     DEFINE_COLUMN(dx1, COLUMN_DX1, "f4", 3);
     DEFINE_COLUMN(dx2, COLUMN_DX2, "f4", 3);
+    DEFINE_COLUMN(dv1, COLUMN_DV1, "f4", 3);
```

Comment:
Did you search the code see if you audited where dx1 and COLUMN_DX1 are mentioned? does dv1 need to be modified similarly?

Reply:
I think I covered everything, except for report_lpt -- not going to report dv1.